entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/impl/BaseEntity.java

```diff
@@ -55,7 +55,7 @@
 
 	// The time at which the Set was modified, after creation. 
 //	protected Date modified;
-	private Aggregation isAggregatedBy;
+	protected Aggregation isAggregatedBy;
```

Comment:
the class attributes are private, or with package visibility if required, but not protected, please revert back

Reply:
Please note that we need to access the BaseEntity fields from its derived classes during the validation of all fields in the ValidEntityFieldsValidator class, which uses the Java reflection. I do not see any problem in letting the fields be protected.